### ISSUE TITLE
Unable to cinfig audio imput with more than 2 channel

### ISSUE THREAD

**Issue by MrMathias154** [ultrastares/ultrastar-worldparty#39]
Hello i search to use with ultrastar-worldparty and audio card {Tascam 16x08} but im unable to configure more than 2 mic imput

Thanks for view this ticket and help me.

**Comment by TeLiXj** [comment]
Hi! At this moment only 2 channels per soundcard are available. You must wait to the next version to use more.

**Comment by jose1711** [comment]
Same issue with Zedi10FX (4 input mic channels, but only 2 are usable).

**Comment by TeLiXj** [comment]
Yes, it will fail in all sound cards with more than 2 channels. As I said in the "real" next version will be fixed, because the 21.02 version is only to fix bugs and add better linux support

**Comment by maxpel** [comment]
I am running into the same issue. Would be great if that functionality could be included!

**Comment by sypten** [comment]
By the way how do you make it work with two mics? I've been trying to do so, but when I plug my singstar adapter I can't open Ultrastar anymore uwu

**Comment by PrettRett** [comment]
If you are in Windows, you need to go to:
_Open Sound Configurations_ -> _Input Section_ -> _Device properties_ -> _aditional properties_ 
And in the advance options, choose one of the 2 channel options

**Comment by MrMathias154** [comment]
> If you are in Windows, you need to go to: _Open Sound Configurations_ -> _Input Section_ -> _Device properties_ -> _aditional properties_ And in the advance options, choose one of the 2 channel options

You didn't understand my problem.
To be clear

My soundcards show only **1** devices, with 16 virtual audio channels

Like in this screenshot, you can view that I am only 2 allowed to select 2 channels per devices, But I have more than 2 channels on it. (Here channel 2 and 4 selected)




> By the way how do you make it work with two mics? I've been trying to do so, but when I plug my singstar adapter I can't open Ultrastar anymore uwu
For you, sometimes I have the same issue is related to permission on USB, try another USB port, sometimes seems to work, and other times not working (issue related to windows).

**Comment by endaro** [comment]
Hi- the multichannel interface is solved?

I am panning buy the UMC1820 to play with the 6 players and only one interface.

**Comment by Gamrin** [comment]
Chiming in on this thread. I have the Behringer UMC1820, and I'm trying to run through proton experimental on NixOS through Lutris.

So far this extraConfig has enabled me to see the microphones and choose them as sides of a stereo. However I've noticed serious lag on mics 5-6

`
extraConfig.pipewire."90-umc1820.conf" = {
      "context.modules" = [
        # === AUX1-2 as stereo virtual mic ===
        {
          "name" = "libpipewire-module-loopback";
          "args" = {
            "node.description" = "UMC1820 Aux1-2 (stereo)";

            "capture.props" = {
              "audio.position" = [ "AUX0" "AUX1" ];
              "stream.dont-remix" = true;
              "node.target" =
                "alsa_input.usb-BEHRINGER_UMC1820_DF74641D-00.multichannel-input";
              "node.passive" = true;
            };

            "playback.props" = {
              "node.name" = "umc1820_aux1_2_stereo";
              "media.class" = "Audio/Source";
              "audio.position" = [ "FL" "FR" ];
              "node.virtual" = true;
            };
          };
        }

        # === AUX3-4 as stereo virtual mic ===
        {
          "name" = "libpipewire-module-loopback";
          "args" = {
            "node.description" = "UMC1820 Aux3-4 (stereo)";

            "capture.props" = {
              "audio.position" = [ "AUX2" "AUX3" ];
              "stream.dont-remix" = true;
              "node.target" =
                "alsa_input.usb-BEHRINGER_UMC1820_DF74641D-00.multichannel-input";
              "node.passive" = true;
            };

            "playback.props" = {
              "node.name" = "umc1820_aux3_4_stereo";
              "media.class" = "Audio/Source";
              "audio.position" = [ "FL" "FR" ];
              "node.virtual" = true;
            };
          };
        }

        # === AUX5-6 as stereo virtual mic ===
        {
          "name" = "libpipewire-module-loopback";
          "args" = {
            "node.description" = "UMC1820 Aux5-6 (stereo)";

            "capture.props" = {
              "audio.position" = [ "AUX4" "AUX5" ];
              "stream.dont-remix" = true;
              "node.target" =
                "alsa_input.usb-BEHRINGER_UMC1820_DF74641D-00.multichannel-input";
              "node.passive" = true;
            };

            "playback.props" = {
              "node.name" = "umc1820_aux5_6_stereo";
              "media.class" = "Audio/Source";
              "audio.position" = [ "FL" "FR" ];
              "node.virtual" = true;
            };
          };
        }

        # === AUX7-8 as stereo virtual mic ===
        {
          "name" = "libpipewire-module-loopback";
          "args" = {
            "node.description" = "UMC1820 Aux7-8 (stereo)";

            "capture.props" = {
              "audio.position" = [ "AUX6" "AUX7" ];
              "stream.dont-remix" = true;
              "node.target" =
                "alsa_input.usb-BEHRINGER_UMC1820_DF74641D-00.multichannel-input";
              "node.passive" = true;
            };

            "playback.props" = {
              "node.name" = "umc1820_aux7_8_stereo";
              "media.class" = "Audio/Source";
              "audio.position" = [ "FL" "FR" ];
              "node.virtual" = true;
            };
          };
        }
      ];
    };
`
